decidim-proposals/app/controllers/decidim/proposals/proposal_endorsements_controller.rb

```diff
@@ -6,6 +6,7 @@ module Proposals
     class ProposalEndorsementsController < Decidim::Proposals::ApplicationController
       helper_method :proposal
 
+      protect_from_forgery except: [:destroy, :create]
```

Comment:
Is this really necessary? It seems that with this another site could trigger a creation or deletion of an endorsement (maybe I'm wrong)

Reply:
This has been added when applying new designs because the CTA is no longer a button with a form, but a whole div that on click executes a jquery method using vanilla $.ajax. This ajax call does not have any authentication_token to be used when invoking :destroy or :create.

I'll try to rework this.

Reply:
I've managed to supply the auth token through the url so now protection from forgery has been re-enabled.